### PR TITLE
Update readme with shields 

### PR DESCRIPTION
**Pull request by RegenJacob**
I added shields.io shields and linked egui in the header

### CODE REVIEW

**Comment by Jerrody** [comment]
Would be nice if to add also the current `egui` version that uses `egui_dock`.

File: README.md [hunk 1]
```diff
@@ -1,4 +1,7 @@
-# `egui_dock`: docking support for `egui`
+[![Crates.io](https://img.shields.io/crates/v/egui_dock)](https://crates.io/crates/egui_dock)
+[![docs.rs](https://img.shields.io/docsrs/egui_dock)](https://docs.rs/egui_dock/)
+
+# `egui_dock`: docking support for [egui](https://github.com/emilk/egui)
 
 Credit goes to [@Iain-dono](https://github.com/lain-dono) for implementing the actual library.
```
Comment (Jerrody) [inline README.md:6]: According to the MD rule [041](https://github.com/DavidAnson/markdownlint/blob/v0.25.1/doc/Rules.md#md041):
> First line in a file should be a top-level heading

Change it to:
```diff
+ # `egui_dock`: docking support for [egui](https://github.com/emilk/egui)
+
+ [![Crates.io](https://img.shields.io/crates/v/egui_dock)](https://crates.io/crates/egui_dock)
+ [![docs.rs](https://img.shields.io/docsrs/egui_dock)](https://docs.rs/egui_dock/)
-
- # `egui_dock`: docking support for [egui](https://github.com/emilk/egui)
```

**Review by Jerrody — CHANGES_REQUESTED** [review]
Please, change highlighted section. After that squash the commits into the new one if it possible.

**Review by Jerrody — APPROVED** [review]

**Comment by RegenJacob** [comment]
Ok i kind of messed my repository up 
would it be ok if i close this pull request and open a new one?
im still a bit new to git

**Comment by Jerrody** [comment]
> Ok i kind of messed my repository up would it be ok if i close this pull request and open a new one? im still a bit new to git

Yeah, why not,

**Pull request closed without merging**